Mathlib/Geometry/Manifold/Instances/Real.lean

```diff
@@ -92,32 +92,36 @@ theorem range_euclideanHalfSpace (n : ℕ) [NeZero n] :
 
 open ENNReal in
 @[simp]
-theorem interior_halfspace {n : ℕ} (p : ℝ≥0∞) (a : ℝ) (i : Fin n) :
+theorem interior_halfSpace {n : ℕ} (p : ℝ≥0∞) (a : ℝ) (i : Fin n) :
```

Comment:
If we rename `halfspace` to `halfSpace`, we should be consistent about it. Can you fix all other mentions of "halfspace" in mathlib as well, please? (At this point, this would make a good separate PR.)

Reply:
Yes, I'll start a new PR just for `halfspace` -> `halfSpace`.